spec/requests/api/v1/users/profile_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe 'GET api/v1/users/profile', type: :request do
+  let(:user) { create(:user) }
+  let(:headers) { auth_headers(user) }
+
+  subject do
+    get api_v1_users_profile_path, headers: headers, as: :json
+    response
+  end
+
+  it { is_expected.to have_http_status(:ok) }
```

Comment:
do you know if `success` the same as `ok`?

Reply:
Yes Leti, it's the same thing. Do you want to use **success** for the test ?

Reply:
no idea, use the standard one if there is one